code/controllers/subsystem/job.dm

```diff
@@ -474,6 +487,9 @@ SUBSYSTEM_DEF(job)
 			if(!job.player_old_enough(player.client))
 				level6++
 				continue
+			if(job.available_in_playtime(player.client))
```

Comment:
available_in_playtime works like available_in_days. 0 means "available now". >0 means "available in ...". IE: not currently available.